frontend/pages/index.vue

```diff
@@ -1,3 +1,132 @@
 <template>
-  <h1>Home Page</h1>
+  <v-container fluid>
+    <v-row>
+      <v-col cols="12">
+        <h1 class="text-h4 ml-4"> Dashboard </h1>
+      </v-col>
+
+
+      <v-toolbar>
+        <v-toolbar-items class="pl-8 align-center">
+          <v-chip/>
+        </v-toolbar-items>
+      </v-toolbar>
+
+      <v-col cols="12" class="px-6">
+        <v-row>
+          <v-col v-for="equipment in equipments" :key="equipment.id" cols="4">
+            <EquipmentCard :equipment="equipment" />
+          </v-col>
+        </v-row>
+      </v-col>
+    </v-row>
+
+
+  </v-container>
 </template>
+
+<script setup lang="ts">
+import type { MockEquipment } from "~/components/equipment-card.vue";
+
+// import type { EquipmentOverviewFragment } from "~/generated/graphql/operations";
+
+
+// const equipments: EquipmentOverviewFragment[] = [];
+const equipments: MockEquipment[] = [
```

Comment:
I recommend making a utility function that can make these mock objects for you, given just a name and the three metrics. That'll lower the amount of code here a lot